cs-minibot-platform-src/src/main/java/minibot/BaseHTTPInterface.java

```diff
@@ -68,6 +76,16 @@ public static void main(String[] args) {
         // Global objects
         JsonParser jsonParser = new JsonParser();
         Gson gson = new Gson();
+        patrolPoints = new ArrayList<>();
+        Course course = new Course();
+        ArrayList<VisionCoordinate> outerTrackCoords = new ArrayList<>();
+        innerTrackCoords = new ArrayList<>();
+        ArrayList<VisionCoordinate> startAreaCoords = new ArrayList<>();
+        ArrayList<VisionCoordinate> middleAreaCoords = new ArrayList<>();
+        advancedAI = new ArrayList<>();
```

Comment:
all of this should _really_ be in its own module as it is completely polluting the http interface, but that will have to be next semester's work.